docs/data-model.rst

```diff
@@ -997,9 +997,22 @@ Consider the following example:
 
 In this tree, node 4 is isolated, and therefore for any sites that are
 on this tree, the state that it is assigned is a special value
-``tskit.MISSING_DATA``, or ``-1``. See the :meth:`TreeSequence.variants`
-method and :class:`Variant` class for more information on how missing
-data is represented.
+``tskit.MISSING_DATA``, or ``-1``. Isolated nodes in a tree can be identified
+using the :meth:`Tree.is_isolated` method. Note that, although isolated, node
+4 above is still considered as being present in the tree. This means, for example,
+that the node is still returned by the :meth:`Tree.nodes` and :meth:`Tree.samples`
+methods. We can therefore identify isolated nodes, as well as nodes linked to the
+main tree topology, when iterating using these methods:
```

Comment:
This is implying that we have one root: we often have many roots in forward simulations. Is there some way to rephrase that's not talking about "the tree topology"?

Reply:
Yep, I was trying to find the right wording for "nodes in one of the conventional trees". Do you think it's reasonable to call these "nodes in a topology" (without implying that there is only one topology per tree), so e.g. 

> We can therefore identify isolated nodes, as well as nodes in a topology,

Reply:
I don't think there's much point in talking about the negative --- isn't it enough to say that we're identifying the isolated nodes? The reader can reasonably infer that they also identify non-isolated nodes, which saves us getting into a confusing technical discussion of what non-isolated nodes are.